k8s.go

```diff
@@ -252,3 +307,98 @@ func secretToString(rawSecret []byte, doubleBase64 bool) (string, error) {
 
 	return content, nil
 }
+
+func getConfigMapK8s(client *kubernetes.Clientset,
+	namespace, name string) (*api.ConfigMap, bool, error) {
+
+	log("Attempting to load configmap %s from namespace %s", name, namespace)
+	configMap, err := client.CoreV1().ConfigMaps(namespace).Get(
+		context.Background(), name, metav1.GetOptions{},
+	)
+
+	switch {
+	case err == nil:
+		log("ConfigMap %s loaded successfully", name)
+		return configMap, true, nil
```

Comment:
nit: in switch statements that aren't just one line per `case`, we like to give ourselves some breathing room and add an empty line before each new `case` or `default`.

Reply:
Updated! ✅